library/pnotes.inc

```diff
@@ -455,13 +456,19 @@ function addMailboxPnote(
     );
 }
 
-function updatePnote($id, $newtext, $title, $assigned_to, $message_status = "")
+function updatePnote($id, $newtext, $title, $assigned_to, $message_status = "", $datetime = "")
 {
     $row = getPnoteById($id);
     if (! $row) {
         die("updatePnote() did not find id '".text($id)."'");
     }
 
+    if (empty($datetime)) {
+        $datetime = date('Y-m-d H:i:s');
+    } else {
+        $datetime = DateTimeToYYYYMMDDHHMMSS($datetime);
```

Comment:
It's really messy to do this conversion this deep in the code. Much better if can do this when collect the variable in the main script.

Reply:
Done